src/ext/node/resolvers.rs

```diff
@@ -31,6 +32,7 @@ use sys_traits::impls::RealSys;
 use super::cjs_translator::{NodeCodeTranslator, RustyCjsCodeAnalyzer};
 
 const NODE_MODULES_DIR: &str = "node_modules";
+const TYPESCRIPT_VERSION: &str = "5.8.3";
```

Comment:
This was lifted from https://github.com/denoland/deno/blob/395878c5fb72d87a44f7c88ce9f391acc31feab3/cli/lib/version.rs

It might need to be kept in sync somehow?